src/ontology/hp-edit.owl

```diff
@@ -201541,66 +202172,72 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://pu
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0100501> "Recurrent bronchiolitis")
 SubClassOf(<http://purl.obolibrary.org/obo/HP_0100501> <http://purl.obolibrary.org/obo/HP_0002837>)
 
-# Class: <http://purl.obolibrary.org/obo/HP_0100502> (Vitamin B12 deficiency)
+# Class: <http://purl.obolibrary.org/obo/HP_0100502> (Decreased circulating vitamin B12 concentration)
 
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0100502> "The concentration of vitamin B12 in the blood circulation is below the lower limit of normal.")
 AnnotationAssertion(dcterms:creator <http://purl.obolibrary.org/obo/HP_0100502> <https://orcid.org/0009-0006-4530-3154>)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/HP_0100502> "2010-12-17T06:00:43Z")
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/HP_0100502> "MSH:D014806")
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/HP_0100502> "SNOMEDCT_US:190634004")
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/HP_0100502> "UMLS:C0042847")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/HP_0100502> "Vitamin B12 deficiency")
 AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasSynonymType> <http://purl.obolibrary.org/obo/hp#layperson>) <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/HP_0100502> "Vitamin B12 deficiency")
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/HP_0100502> "HP:0100502")
-AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0100502> "Vitamin B12 deficiency")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0100502> "Decreased circulating vitamin B12 concentration")
```

Comment:
This label change seems a bit lossy - but I get it that you expect people to know that when they read an HPO label, they will think "it must be abnormally decreased"; not all forms of decreased concentration are abnormal

Reply:
I am trying to standardize the term labels for laboratory. The definitions also need to be standardized across the entire ontology. The original label was simply wrong.